src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -624,102 +640,113 @@ export const TradeInput = ({ isCompact }: TradeInputProps) => {
               visibility={isCompactQuoteListOpen ? 'hidden' : undefined}
               position={isCompactQuoteListOpen ? 'absolute' : undefined}
             >
-              <WarningAcknowledgement
-                message={warningAcknowledgementMessage}
+              <StreamingAcknowledgement
                 onAcknowledge={handleFormSubmit}
-                shouldShowWarningAcknowledgement={shouldShowWarningAcknowledgement}
-                setShouldShowWarningAcknowledgement={setShouldShowWarningAcknowledgement}
+                shouldShowAcknowledgement={shouldShowStreamingAcknowledgement}
+                setShouldShowAcknowledgement={setShouldShowStreamingAcknowledgement}
+                estimatedTime={
```

Comment:
suggestion: this could be `T | undefined for type-safety, so that `StreamingAcknowledgement` can ensure we never end up in a state where the time estimation is undefined

Reply:
Disregard me - just gave it a try locally and this rugs whichever component is wrapped in ack